Userland/Libraries/LibJS/Runtime/Object.h

```diff
@@ -232,6 +233,9 @@ class Object : public Cell {
 
     Object const* prototype() const { return shape().prototype(); }
 
+    // https://w3c.github.io/webappsec-csp/#get-csp-of-object
+    Web::HTML::CSP_List* retrieve_the_csp_list_of_an_object();
```

Comment:
Adding Web:: namespaced things to LibJS is a layering violation. LibWeb depends on LibJS, not the other way around.

If you want this method to take a generic object, it should be a free function declared in PolicyContainers.h or some such that takes a JS::Object& and can check `is<HTML::Document>(object)`

Reply:
I think the conflict is resolved, but maybe where I ended up putting that function isn't entirely appropriate. It might also become clearer where to put it as more of that spec is implemented, so whether or not to move it more now is your call